ouroboros-consensus-cardano/tools/db-analyser/Analysis.hs

```diff
@@ -319,8 +320,30 @@ storeLedgerStateAt slotNo AnalysisEnv { db, backing, initLedger, cfg, ledgerDbFS
       Right (ldb, _) -> do
         -- The replay performed above leaves the DbChangelog at @tip - k@
         -- blocks. We use this flush to push all the remaining differences into
-        -- the backing store.
-        OnDisk.flush backingStore $ ledgerDbChangelog ldb
+        -- the backing store. First, we must update the @DbChangeLog@ such that
+        -- it reflects the fact that we are flushing all remaining differences.
+
```

Comment:
Maybe expand a bit on this.

Reply:
My latest commit should expand on this, and include the TODO.